webapp/src/ts/modals/edit-user/update-password.component.ts

```diff
@@ -23,11 +23,18 @@ export class UpdatePasswordComponent {
   processing = false;
   errors: any;
   editUserModel: {
-    username?;
-    currentPassword?;
-    password?;
-    passwordConfirm?;
-  } = {};
+    username?: string;
+    currentPassword?: string;
+    password?: string;
+    passwordConfirm?: string;
+    passwordFieldType: string;
+    showPasswordIcon: string;
+    hidePasswordIcon: string;
+  } = {
+      passwordFieldType: 'password', // Start with the password masked.
+      showPasswordIcon: '/login/images/show-password.svg', // Icon for showing password.
+      hidePasswordIcon: '/login/images/hide-password.svg' // Icon for masking password.
```

Comment:
```suggestion
      passwordFieldType: 'password',
      showPasswordIcon: '/login/images/show-password.svg',
      hidePasswordIcon: '/login/images/hide-password.svg',
```

These are descriptive enough, no need to include the comments. 

Also, please define them as constants at the top of the file and use the constants here like you did in the Admin app